.gitignore

```diff
@@ -28,3 +28,5 @@ yarn-error.log*
 !.yarn/versions
 
 .vercel
+
+.idea
```

Comment:
interesting! but fine with me

Reply:
I am using RustRover (based on IntelliJ IDEA) for everything. :)